cmd/junitreport/pkg/cmd/junitreport.go

```diff
@@ -0,0 +1,115 @@
+package cmd
+
+import (
+	"bufio"
+	"encoding/xml"
+	"fmt"
+	"io"
+
+	"github.com/openshift/origin/cmd/junitreport/pkg/builder"
+	"github.com/openshift/origin/cmd/junitreport/pkg/builder/flat"
+	"github.com/openshift/origin/cmd/junitreport/pkg/builder/nested"
+	"github.com/openshift/origin/cmd/junitreport/pkg/parser"
+	"github.com/openshift/origin/cmd/junitreport/pkg/parser/gotest"
+)
+
+type testSuitesBuilderType string
+
+const (
+	flatBuilderType   testSuitesBuilderType = "flat"
```

Comment:
Since you follow conventions from the main repo, I would expect these constants under your api package.